src/components/Comments/ReplyCommentItem.js

```diff
@@ -158,16 +162,28 @@ const ReplyCommentItem = ({
       onLongPress();
     }
   };
-
   const username = comment?.data?.anon_user_info_color_name
-    ? `Anonymous ${comment.data?.anon_user_info_emoji_name}`
+    ? `${comment.data.anon_user_info_color_name} ${comment.data?.anon_user_info_emoji_name}`
     : user?.data?.username;
 
+  const onPressDm = async () => {
+    try {
+      refSheet.current.open();
+      setLoading({...loading, loadingGetAllowAnonDmStatus: true});
+      const data = await getAllowAnonDmStatus('comment', comment?.id);
+      setUserAllowDm(data?.user.allow_anon_dm);
+    } catch (e) {
+      console.log(e);
+    } finally {
+      setLoading({...loading, loadingGetAllowAnonDmStatus: false});
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `onPressDm` has been updated to fetch whether the user allows anonymous DMs or not. It's good to see error handling and loading state management in place. However, consider adding a more descriptive error message instead of just logging the error to the console. This will help with debugging if something goes wrong.

```diff
- console.log(e);
+ console.error("Error fetching allow anonymous DM status: ", e);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->